package.json

```diff
@@ -9,12 +9,12 @@
     "e2e-headful": "wireit",
     "e2e-headless": "wireit",
     "e2e": "npm run e2e-headless --",
-    "eslint": "eslint --ext js --ext mjs --ext ts --fix .",
     "flake8": "flake8 examples/ tests/",
     "format": "npm run pre-commit --",
+    "format:eslint": "eslint --ext js --ext mjs --ext ts --fix .",
```

Comment:
Convetion for Package json - https://eslint.org/docs/latest/contribute/package-json-conventions
We should consider using it for the other commands as well.